polyply/src/gen_coords.py

```diff
@@ -115,6 +115,7 @@ def gen_coords(toppath,
                step_fudge=1.0,
                max_force=5*10**4.0,
                nrewind=5,
+               lib=None,
```

Comment:
I don't dislike this but also not a fan still ... We'll see

Reply:
By default the force_field as the name of the molecule that is generated. As far as I know it is not possible to get the name of the force field from the actual force_field attribute of the Topolgy, right?

Because in that case we can just read it from the force field.

Reply:
this would be the best solution but I think there is no universal way to do this

Reply:
@jan-stevens please open this as an issue for improvement